Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should be in the document when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/4/2e869eb4-999a-43a8-a2c8-38827ef7cd7d.html b/output/Checkbox Group with Validation Error/4/2e869eb4-999a-43a8-a2c8-38827ef7cd7d.html
index 57460f8..444bcdf 100644
--- a/output/Checkbox Group with Validation Error/4/2e869eb4-999a-43a8-a2c8-38827ef7cd7d.html	
+++ b/output/Checkbox Group with Validation Error/4/2e869eb4-999a-43a8-a2c8-38827ef7cd7d.html	
@@ -1,18 +1,16 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <div>
-      <input type="checkbox" id="bananas" name="fruits" value="bananas">
-      <label for="bananas">Bananas</label>
-    </div>
-    <div>
-      <input type="checkbox" id="oranges" name="fruits" value="oranges">
-      <label for="oranges">Oranges</label>
-    </div>
-    <div>
-      <input type="checkbox" id="apples" name="fruits" value="apples">
-      <label for="apples">Apples</label>
-    </div>
-    <div style="color:red;display:none;" id="error">You must choose one or more fruits</div>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset aria-describedby="error">
+  <legend>What fruits do you like?</legend>
+  <div id="error">You must choose one or more fruits</div>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
